downstream/modules/platform/ref-ports-firewalls-requirements.adoc

```diff
@@ -1,11 +1,11 @@
 
-[id="ref-access-control-requirements_{context}"]
+[id="ref-ports-firewalls-requirements_{context}"]
 
-= Access control requirements
+= Ports and firewalls requirements
```

Comment:
This title seems a little off to me tbh. Ports and firewalls are separate but related things. What about something like "Network ports and protocols" or maybe "Network traffic requirements"?